x-pack/plugins/security_solution/server/endpoint/routes/metadata/handlers.ts

```diff
@@ -54,6 +56,8 @@ export function getMetadataListRequestHandler(
         total,
         page: request.query.page || ENDPOINT_DEFAULT_PAGE,
         pageSize: request.query.pageSize || ENDPOINT_DEFAULT_PAGE_SIZE,
+        sortField: request.query.sortField || ENDPOINT_DEFAULT_SORT_FIELD,
```

Comment:
that's a good question! I followed the AC in the original issue to use the existing sorting by default, because I found it reasonable to not change the existing behaviour, but extend it with the possibility to sort by other fields. but I agree, it's a bit cumbersome to remove the sorting.

could you help, @caitlinbetz? the problem is that the enrollment date used for default sorting is not displayed in the table, so the user cannot go back to the default sorting easily. what do you think is better for users? keeping the original sorting they are used to (but making it hard to return to), or changing the default sorting, so it's made based on a displayed field?

with a bit more effort, we also can solve this by adding a new column for the 'enrolled' field, but this makes the already packed table even denser, or adding a 'clear sorting' button but that's a bit weird

Reply:
I think UX wise it is best that if we're introducing sorting on the table then we start with a sorted column on the table that clearly informs the user what is sorted, and not sort with an invisible UX value. Using `enrolled at` to sort made sense earlier as we were not able to sort any column; with this change I think it is obsolete.